Doc: update `README.md` with example showing how to use HTTPListener (#28)
Dynatrace/dynatrace-configuration-as-code-core: README.md

resp, err := ctx.Get(ctx,"...")
```

### Tracking and logging HTTP requests/responses

> If you want to keep track or just log all HTTP requests/responses that are happening as part of the execution of the clients you can attach an `HTTPListener` that can be used

Reviewer: ```suggestion
If you want to keep track or just log all HTTP requests/responses happening as part of the execution of the clients, you can implement an `HTTPListener` and attach it to the client.
```
